fe/fe-core/src/main/java/org/apache/doris/common/util/TimeUtils.java

```diff
@@ -85,32 +84,42 @@ public class TimeUtils {
     public static Date MIN_DATETIME = null;
     public static Date MAX_DATETIME = null;
 
-    private static ThreadLocal<SimpleDateFormat> datetimeFormatThreadLocal =
-            ThreadLocal.withInitial(() -> new SimpleDateFormat("yyyy-MM-dd HH:mm:ss"));
+    public static final DateTimeFormatter DATETIME_MS_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSS")
+            .withZone(ZoneId.systemDefault());
 
-    private static ThreadLocal<SimpleDateFormat> datetimeMSFormatThreadLocal =
-            ThreadLocal.withInitial(() -> new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS"));
+    public static final DateTimeFormatter DATETIME_NS_FORMAT = DateTimeFormatter.ofPattern(
+                    "yyyy-MM-dd HH:mm:ss.SSSSSSSSS")
+            .withZone(ZoneId.systemDefault());
+
+    public static final DateTimeFormatter DATETIME_FORMAT_WITH_HYPHEN = DateTimeFormatter.ofPattern(
+                    "yyyy-MM-dd-HH-mm-ss")
+            .withZone(ZoneId.systemDefault());
 
     static {
-        TIME_ZONE = new SimpleTimeZone(8 * 3600 * 1000, "");
+        TIME_ZONE = ZoneId.of("UTC+8");
 
-        DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd");
-        DATE_FORMAT.setTimeZone(TIME_ZONE);
+        DATE_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+        DATE_FORMAT.withZone(TIME_ZONE);
 
-        DATETIME_FORMAT = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
-        DATETIME_FORMAT.setTimeZone(TIME_ZONE);
+        DATETIME_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+        DATETIME_FORMAT.withZone(TIME_ZONE);
 
-        TIME_FORMAT = new SimpleDateFormat("HH");
-        TIME_FORMAT.setTimeZone(TIME_ZONE);
+        TIME_FORMAT = DateTimeFormatter.ofPattern("HH");
+        TIME_FORMAT.withZone(TIME_ZONE);
 
         try {
-            MIN_DATE = DATE_FORMAT.parse("0000-01-01");
-            MAX_DATE = DATE_FORMAT.parse("9999-12-31");
 
-            MIN_DATETIME = DATETIME_FORMAT.parse("0000-01-01 00:00:00");
-            MAX_DATETIME = DATETIME_FORMAT.parse("9999-12-31 23:59:59");
+            MIN_DATE = Date.from(
+                    LocalDate.parse("0001-01-01", DATE_FORMAT).atStartOfDay().atZone(TIME_ZONE).toInstant());
+            MAX_DATE = Date.from(
+                    LocalDate.parse("9999-12-31", DATE_FORMAT).atStartOfDay().atZone(TIME_ZONE).toInstant());
+
+            MIN_DATETIME = Date.from(
```

Comment:
MIN_DATETIME is never used

Reply:
MIN_DATE and MIN_DATETIME are only used in TimeUtilsTest